hybrisprovider.cpp

```diff
@@ -352,6 +352,15 @@ int HybrisProvider::GetStatus()
     return m_status;
 }
 
+int HybrisProvider::GnssDeleteAidingData()
+{
+    if (m_backend) {
+        m_backend->gnssDeleteAidingData(0xFFFF);
```

Comment:
can you add a code comment for why 0xFFFF is used

Reply:
Removes all data

Reply:
Can you please also add it as a code comment, to document it for future readers of the code?

Reply:
Fix it